sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -169,6 +172,30 @@ def test_do_with_multiple_outputs_maintains_unique_name(self):
     pipeline.run()
 
   @attr('ValidatesRunner')
+  def test_read_from_text_metrics(self):
+    from apache_beam.examples.snippets.snippets import CountingSource
```

Comment:
Agreed. This is not clean. The solution here is to copy `CountingSource` or maybe make `CountingSource` a first class test class in the SDK. Could you file a JIRA issue for this?